### ISSUE TITLE
[2.8.3] EOL spring-boot component in use

### ISSUE THREAD

**Issue by tuxmaster5000** [obiba/agate#524]
In version 2.8.3 in eol version of the framework is used.
Please update it to an current one.
Report of the security scanner:
```
The "VMware Spring Boot" version on the remote host has reached the end of life.

CPE:               cpe:/a:vmware:spring_boot:2.7.8
Installed version: 2.7.8
Location/URL:      /usr/share/agate-2.8.3/webapp/WEB-INF/lib/spring-boot-2.7.8.jar
EOL version:       2.7
EOL date:          2023-11-18
```

**Comment by tuxmaster5000** [comment]
See more:
https://spring.io/projects/spring-boot#support

**Comment by ymarcon** [comment]
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide

**Comment by ymarcon** [comment]
Available dev version with Java 17, Spring Boot 3.1, Spring 6.0:
* Docker tag obiba/agate:snapshot
* Online https://agate-test.obiba.org/

**Comment by ymarcon** [comment]
Release available 3.0.0-beta1, requires java 21
